pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: layout.pug
+navigationTitle: Application Deployment
+title: Deployment of Applications
+menuWeight: 5
+beta: false
+excerpt: Deploy applications to attached clusters using the CLI
+---
+
+<!-- markdownlint-disable MD004 MD040 -->
+
+This topic describes how to use the CLI to deploy a workspace catalog application to attached clusters within a workspace.
+
+## Prerequisites
+
+Before you begin, you must have:
+
+- A running cluster with [Kommander installed](../../../../install/).
+- An [existing Kubernetes cluster attached to Kommander](../../../../clusters/attach-cluster/).
+
+Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace's namespace where the cluster is attached:
+
+```sh
+export WORKSPACE_NAMESPACE=<workspace_namespace>
+```
+
+## Deploy the application
+
+The list of available applications that can be deployed on the attached cluster can be found [in this documentation](../../catalog-applications#workspace-catalog-applications).
```

Comment:
```suggestion
Find the list of available applications that can be deployed on the attached cluster [in this documentation](../../catalog-applications#workspace-catalog-applications).
```

Reply:
going to use Casie's suggestion above for this one since that one also eliminates "be deployed"

Reply:
done